contrib/epee/include/net/levin_protocol_handler_async.h

```diff
@@ -352,13 +351,13 @@ class async_protocol_handler
       MERROR(m_connection_context << "[levin_protocol] -->> start_outer_call failed");
       return false;
     }
-    boost::interprocess::ipcdetail::atomic_inc32(&m_wait_count);
+    m_wait_count++;
```

Comment:
Not sure if it needs to be changed but... I had to look up whether post-incrementing an atomic int had any performance implications (like it does for iterators). (it doesn't, but someone else might have the same question needlessly)

Reply:
Does postfix for iterators take a hit mainly because it has to copy construct itself? If so, it would make sense that `std::atomic<uint32_t>` wouldn't have that issue (except for single 4 byte `mov` instruction`, which is probably optimized away anyways)

Reply:
The main problem here is cognitive load with no functional benefit. Best to do the simplest thing possible, which is a pre-increment.

Reply:
Is a postfix increment harder to understand than a prefix increment?

Reply:
Doing more things = more things to think about, that's all.